assembl/static2/js/app/components/common/termsForm.jsx

```diff
@@ -79,4 +98,6 @@ const withData = graphql(LegalNoticeAndTerms, {
   }
 });
 
-export default compose(withData, withLoadingIndicator())(TermsForm);
```

Comment:
please add flow types for `withData` function too (see what I did in `legalNotice.jsx` for example), it wont take long I promise ;-)